rspace/src/test/scala/coop/rchain/rspace/history/HistoryRepositoryGenerativeSpec.scala

```diff
@@ -46,13 +46,19 @@ class LMDBHistoryRepositoryGenerativeSpec
     )
 
   def lmdbHistoryStore =
-    HistoryStoreInstances.historyStore(StoreInstances.lmdbStore[Task](lmdbConfig))
+    HistoryStoreInstances.historyStore(
+      StoreInstances.lmdbStore[Task](lmdbConfig).runSyncUnsafe(Duration.Inf)
```

Comment:
Is it a lot of effort to get rid of the `runSyncUsafe`s?

Reply:
Done